src/modals/storage-connections/StorageConnectionsModal.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import PluginApiModal from '_/components/modals/PluginApiModal'
+import getPluginApi from '_/plugin-api'
+import { msg } from '_/intl-messages'
+import { Spinner } from 'patternfly-react'
+import { Button } from '@patternfly/react-core'
+import StorageConnectionsModalBody from './StorageConnectionsModalBody'
+import { createErrorMessage } from '_/utils/error-message'
+import './connections.css'
+
+const StorageConnectionsModal = ({
+  isLoading = false,
+  storageDomain,
+  domainConnections,
+  allConnections,
+  onClose = () => { },
```

Comment:
the callbacks seem required properties - if yes they don't require defaults (but should be marked in proptypes as required)

Reply:
Ack